.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Delivery
```

Comment:
Any references to put in the second commit message?

Reply:
Updated the commit message.